ember-cli-flash/src/flash/object.js

```diff
@@ -40,25 +40,24 @@ export default class FlashObject extends EmberObject.extend(Evented) {
       return;
     }
     this.exitTimerTask();
-    this.trigger('didExitMessage');
   }
 
-  willDestroy() {
+  destroy() {
     if (this.onDestroy) {
       this.onDestroy();
     }
 
     this._cancelTimer();
     this._cancelTimer('exitTaskInstance');
-    super.willDestroy(...arguments);
+    this.isDestroyed = true;
   }
```

Comment:
Renamed `willDestroy` hook to `destroy` so this object is sort of pretending to be an Ember object still (the tests check `isDestroyed`.

Not sure if there's operations that happen in the real `destroy()` that we need to handle here.

Reply:
> ❓ Not sure if there's operations that happen in the real destroy() that we need to handle here.

nah, with `@ember/destroyable` the destroy callback can be a standalone function from anywhere